src/us/kbase/common/executionengine/CallbackServer.java

```diff
@@ -241,9 +241,12 @@ protected void processRpcCall(RpcCallData rpcCallData, String token, JsonServerS
         final ModuleMethod modmeth = new ModuleMethod(
                 rpcCallData.getMethod());
         final Map<String, Object> jsonRpcResponse;
-        
+
+
         if (modmeth.isCheck()) {
             jsonRpcResponse = runCheck(rpcCallData);
+            String logString = String.format("BEGIN_IS_CHECK_LOG|RPCDATA:%s|RPCRESPONSE:%s|TIMESTAMP:%s|END_IS_CHECK_LOG", rpcCallData.toString(), jsonRpcResponse.toString(),System.currentTimeMillis());
```

Comment:
This is going to be really hard to read - I'd probably pretty print the json on each line.

Reply:
wait that won't work due to interleaving

Reply:
I'd at least put the different json strings on different lines, using the job id to link up the lines.

Reply:
Yea it looks pretty bad just checked it, will try a nicer format